.cirrus.yml

```diff
@@ -4,12 +4,11 @@ task:
   name: FreeBSD
   env:
     matrix:
-      - JULIA_VERSION: 1.0
```

Comment:
Let's keep a 1.0 build

Reply:
Sorry. I removed these for testing and forgot to put them back.

Reply:
No worries!